lib/std/zig/Ast.zig

```diff
@@ -458,6 +458,16 @@ pub fn renderError(tree: Ast, parse_error: Error, stream: anytype) !void {
             return stream.writeAll("for input is not captured");
         },
 
+        .unterminated_literal => {
+            return stream.print("unterminated '{s} literal'", .{
+                switch (tree.source[tree.tokens.items(.start)[parse_error.token]]) {
+                    '\'' => "character",
+                    '"', '\\' => "string",
```

Comment:
```suggestion
                    '"' => "string",
```
Multiline string literals can't be unterminated.